web/src/engine/websites/ProjectSuki.ts

```diff
@@ -0,0 +1,72 @@
+import { Tags } from '../Tags';
+import icon from './ProjectSuki.webp';
+import { DecoratableMangaScraper, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchWindowScript } from '../platform/FetchProvider';
+
+const pageScript = `
+    new Promise ((resolve, reject) => {
+        try {
+
+            const element = document.querySelector('.strip-reader');
+            const bookid = window.location.href.split('/') [4];
+            const chapterid = window.location.href.split('/') [5];
+            const body = JSON.stringify({
+                bookid: bookid,
```

Comment:
Can be simplified since JSON field and variable name are identical `{ bookid, chapterid, first: true }`